Movies_mvp/Movies_mvp/MovieDetailViewController/Presenter/MovieDetailPresenter.swift

```diff
@@ -0,0 +1,67 @@
+// MovieDetailPresenter.swift
+// Copyright © RoadMap. All rights reserved.
+
+/// Презентер детального описания фильма
+final class MovieDetailPresenter: MovieDetailPresenterProtocol {
+    // MARK: - Public Properties
+
+    weak var view: MovieDetailViewProtocol?
+    var router: RouterProtocol?
+    var movieDetail: MovieDetail?
+    var trailers: [Video] = []
+    var actors: [Actor] = []
+
+    // MARK: - Private Properties
+
+    private let networkService: NetworkServiceProtocol
+    private var movieId = 0
+
+    // MARK: - Initializers
+
+    init(view: MovieDetailViewProtocol?, networkService: NetworkServiceProtocol, id: Int, router: RouterProtocol) {
+        self.view = view
+        self.networkService = networkService
+        self.router = router
+        movieId = id
+    }
+
+    // MARK: - Public methods
+
+    func fetchDetails() {
```

Comment:
```suggestion
    func fetchMovieDetails() {
```
Это же детали фильма а не просто детали